playwright/navigation.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { expect, test } from '@playwright/test';
+
+test('Navigate and check content', async ({ page }) => {
+  await page.goto('/');
+  await expect(page.getByTestId('myHero')).toBeVisible();
+  await expect(page.getByTestId('about')).toBeVisible();
+  await expect(page.getByTestId('portfolio')).toBeVisible();
+  await expect(page.getByTestId('footer')).toBeVisible();
+
+  await page.getByRole('link', { name: 'Portfolio' }).click();
+  await page.waitForURL('/#portfolio');
+  await expect(page.getByTestId('portfolio')).toBeInViewport();
```

Comment:
The `.toBeInViewport()` assertion checks whether the element with `the data-test-id` attribute is visible in the current viewport.